src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Pagination, Navigation } from "swiper/modules";
+import SwiperCore from "swiper";
+import "./Slider.css";
+
+import "swiper/css";
+import "swiper/css/bundle";
+
+SwiperCore.use([Navigation, Pagination]);
+
+const Slider = ({ records }) => {
+  const [windowWidth, setWindowWidth] = React.useState(window.innerWidth);
+
+  React.useEffect(() => {
+    function watchWidth() {
+      setWindowWidth(window.innerWidth);
```

Comment:
Being consistent with how you are using functions. Everythign else is a fat arrow, so change this to => without the function name for consistency.

Reply:
completed